flight/targets/aq32/board-info/board-info.mk

```diff
@@ -1,12 +1,12 @@
 BOARD_TYPE          := 0x94
 BOARD_REVISION      := 0x01
-BOOTLOADER_VERSION  := 0x87
+BOOTLOADER_VERSION  := 0x88
```

Comment:
Would require a rebase and it's only been at 0x88 for a couple of days... will just ignore ;)